escrow/payment_channel_storage.go

```diff
@@ -27,7 +28,8 @@ func NewPaymentChannelStorage(atomicStorage AtomicStorage) *PaymentChannelStorag
 		delegate: &TypedAtomicStorageImpl{
 			atomicStorage: &PrefixedAtomicStorage{
 				delegate:  atomicStorage,
-				keyPrefix: "/payment-channel/storage",
+				//Add the Network as the prefix on the key for storage
```

Comment:
What happens if BlockChainNetworkSelected is not set in the config? If I remember correct user can set the registry address directly and not need to set this correct?

Reply:
the configurtion is now forced and defualt value has been removed , without this config Daemon will not start

Reply:
We shouldnt allow the registryaddress and the networkname to be provided as that can lead to incorrect scenarios.

Reply:
@raamb as discussed , we will now use the MPE address as the storage prefix